### PR TITLE
Tea heals a bit more if you're british

### PR DESCRIPTION
**Pull request by GDNgit**
## What Does This PR Do
Tea heals a bit more if you have the chav mutation
20% chance to heal 1 brute and 1 burn

## Why It's Good For The Game
It's just one of those silly things that I'm honestly surprised isn't in the code yet. Chav is already a total meme and this is just kinda funny. It is chewsday innit so a good day to PR this

## Testing
Compiled, ran, became british, drake tea, got stabbed

## Changelog
:cl:
tweak: Tea heals a bit more if you have the chav mutation
/:cl:

### CODE REVIEW

**Comment by Ionward** [comment]
>  became british, drake tea,

drake tea is what you get when you stick the ashdrake in the pool 🤡

**Comment by Leanfrog** [comment]
INB4 all the powergamers suddenly turn Chav for the free gamer juice. /s

**Comment by GDNgit** [comment]
> INB4 all the powergamers suddenly turn Chav for the free gamer juice. /s

Being British is a strong enough nerf to deal with that. Completely nukes NT dental care 😔

**Comment by ArcZelf** [comment]
God save the king

**Comment by Christasmurf** [comment]
RULE BRITANNIA
BRITANNIA RULES THE WAVES

**Comment by SynthTwo** [comment]
this is a perk that british ppl have irl

File: code/_globalvars/traits.dm [hunk 1]
```diff
@@ -54,6 +54,7 @@ GLOBAL_LIST_INIT(traits_by_type, list(
 		"TRAIT_STURDY_LIMBS" = TRAIT_STURDY_LIMBS,
 
 		"TRAIT_COMIC_SANS" = TRAIT_COMIC_SANS,
+		"TRAIT_CHAV"	   = TRAIT_CHAV,
```
Comment (Sirryan2002) [inline code/_globalvars/traits.dm:57]:
```suggestion
		"TRAIT_CHAV" = TRAIT_CHAV,
```

**Review by Sirryan2002 — APPROVED** [review]

**Review by hal9000PR — APPROVED** [review]
We need to add dragon soop to ss13

**Pull request merged**